benchmarks/file_server/vixeny.ts

```diff
@@ -0,0 +1,23 @@
+import vixeny from "https://deno.land/x/endofunctor@v0.0.62/fun.ts";
+import { PROTOCOL_HTTP_URL_PORT, FILE_SERVER_PATH } from "../SERVER_DATA.ts";
+
+export const NAME = "Vixeny";
+export const DESCRIPTION = "";
+export const VERSION = "0.0.62";
+
+
+if (import.meta.main) {
+  await Deno.serve(
+    vixeny({ hasName: PROTOCOL_HTTP_URL_PORT + "/"})([
+      {
+        type: "static",
+        path: FILE_SERVER_PATH + "/" ,
+        name: "/",
+        mime: false
```

Comment:
Is there a reason you disable `mime` here?
It would be unfair to other benchmarks if this doesn't send `content-type` with proper mimetype.

Reply:
//I don't know if hono adds content type but I'll add it to be as fair as possible

Reply:
Yes it does, [look here](https://github.com/honojs/hono/blob/09175e1289e3b0783b23c8dc3b422d36b4ddcde0/src/adapter/deno/serve-static.ts#L47)